testsuite/preview/expansion/src/test/java/org/wildfly/test/integration/observability/opentelemetry/OpenTelemetryIntegrationWithVertxTestCase.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.integration.observability.opentelemetry;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.testcontainers.api.DockerRequired;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.test.shared.observability.setuptasks.OpenTelemetryWithCollectorSetupTask;
+import org.jboss.as.test.shared.observability.setuptasks.ServiceNameSetupTask;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+/**
+ * A test on opentelemetry to use the Vertx instance defined in the vertx subsystem.
+ * There will be a server log to indicate the usage.
+ */
+@RunWith(Arquillian.class)
+@ServerSetup({OpenTelemetryWithCollectorSetupTask.class, ServiceNameSetupTask.class, OpenTelemetryIntegrationWithVertxTestCase.LoggingWithVertxServerSetupTask.class, VertxSubsystemSetupTask.class})
```

Comment:
If you're not explicitly checking for a particular service name in a trace, the `ServiceNameSetupTask` is not needed here.

Reply:
OK, removed the usage of ServiceNameSetupTask, and use the name of the archive for the tests.

Actually checking for traces is not the main goal for this test case, it tries to send rest request to trigger the tracer so that the necessary messages to be checked will be in the log files.